ecosystem/manager.py

```diff
@@ -324,7 +324,10 @@ def _run_python_tests(
         repo_url: str,
         tier: str,
         python_version: str,
-        package_to_check: str,
+<<<<<<< HEAD
+=======
+        qiskit: str,
+>>>>>>> 2b284cffea465cb413ac35b655efce9f5be03546
```

Comment:
Hey, it looks like you have a lot of merge conflicts still like this, if you look through the "Files changed" tab on GitHub.

Thanks for your contribution by the way! We appreciate your help.

Reply:
Hey, sorry for this..actually i'm a beginner here..learning and applying things. So I made this mistake, so sorry for that...will push the pr again in couple of minutes...Thanks for providing me this opportunity bro.